.changeset/form-footer-split-buttons.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/form-footer': minor
+---
+
+[LG-5039](https://jira.mongodb.org/browse/LG-5039): `backButtonProps`, `cancelButtonProps`, and `primaryButtonProps` can be used to render `SplitButton` instances and make `primaryButtonProps` required
```

Comment:
Would this be a `major` change since we're making an optional prop required? The consumer may not have been using it before, but now they must provide it.